test/e2e/upgrades/apps/job.go

```diff
@@ -17,8 +17,14 @@ limitations under the License.
 package upgrades
 
 import (
+	"fmt"
+	"strings"
+
 	batchv1 "k8s.io/api/batch/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	clientset "k8s.io/client-go/kubernetes"
```

Comment:
The PR job is failed like:
```
Execution platform: //:rbe_with_network
compilepkg: missing strict dependencies:
	/b/f/w/test/e2e/upgrades/apps/job.go: import of "k8s.io/client-go/kubernetes"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.
```
Need to run `./hack/update-bazel.sh` and apply the change into this PR.

Reply:
sry, fixed. thanks a lot for @oomichi help! Could you help to review again?